var/spack/repos/builtin/packages/moab/package.py

```diff
@@ -38,6 +38,8 @@ class Moab(AutotoolsPackage):
     homepage = "https://bitbucket.org/fathomteam/moab"
     url = "http://ftp.mcs.anl.gov/pub/fathom/moab-5.0.0.tar.gz"
 
+    version('5.0.2', '00a6f96f2e6591ab087548839fa3825e',
+             url='ftp://ftp.mcs.anl.gov/pub/fathom/moab-5.0.2.tar.gz')
```

Comment:
you should not need to specify `url`, should be picked up by Spack just fine.

Reply:
```
version('5.0.2', '00a6f96f2e6591ab087548839fa3825e')
```
should be enough, Spack should figure out the right `url` from ` url = "http://ftp.mcs.anl.gov/pub/fathom/moab-5.0.0.tar.gz"`.

Reply:
Thanks. I've tested this, it works. Spack magically found right tarball.